packages/TelegramClient-UI.package/TCUEmojiHelper.class/class/addSingle.to..st

```diff
@@ -0,0 +1,8 @@
+unicode matching
+addSingle: aHexString to: aCollection
```

Comment:
I find the name to be somewhat meaningless. Maybe `add: aHexString intoSubCollectionAtEndOf: aCollection`?

Reply:
Is `append: aHexString toSubcollectionOf: aCollection` ok?